### ISSUE TITLE
Which config is used by OWTB in the final experiment?

### ISSUE THREAD

**Issue by lalalafloat** [YangLiu14/Open-World-Tracking#10]
Hello, By disturb you again. May I ask if the final experiment result of OWTB comes from the config "Detectron2-OWT/Configs/New_BASELINES/MASK_RCNN_R_101_FPN_400EP_LSJ_OWT.PY" ?

**Comment by YangLiu14** [comment]
No, it’s the other one, Panoptic R101. I recently update the readme. So you will find that information in <Data and Model> section :-)
